src/docs/stories/introduction.stories.mdx

```diff
@@ -111,55 +111,6 @@ In order to use style guide components, you have to include CSS file in the `<he
   code={`<link href='https://styleguide.brainly.com/${version}/style-guide.css' rel='stylesheet'/>`}
 />
 
-## Icons (SVG sprites)
```

Comment:
I'd add some info about icons in the introduction

Reply:
What info?